molecule_vmware/cookiecutter/{{cookiecutter.molecule_directory}}/{{cookiecutter.scenario_name}}/create_windows_instance.yml

```
@@ -38,6 +38,21 @@
   delay: 10
   retries: 300
 
+- name: Check vmware tools status
+  vmware_guest_info:
+    hostname: "{{ molecule_yml.driver.vcenter_hostname }}"
+    username: "{{ molecule_yml.driver.vcenter_username }}"
+    password: "{{ molecule_yml.driver.vcenter_password }}"
+    validate_certs: "{{ molecule_yml.driver.validate_certs | default(false) }}"
+    datacenter: "{{ molecule_yml.driver.datacenter }}"
```

```suggestion
    datacenter: "{{ molecule_yml.driver.datacenter }}"
    folder: "{{ molecule_yml.driver.folder }}"
```

`folder` パラメーターを追加してもらえますか？

---

Can you please add the `folder` parameter?

---

追加しました！確かに必要ですね。